lls/nd.c

```diff
@@ -469,6 +478,10 @@ process_nd_neigh_solicitation(struct lls_config *lls_conf, struct rte_mbuf *buf,
 
 		icmpv6_hdr->cksum = rte_ipv6_icmpv6_cksum(ipv6_hdr, icmpv6_hdr);
 
+		ret = adjust_shift_l2_hdr(iface, buf, eth_hdr);
+		if (ret < 0)
+			return -1;
```

Comment:
As my comment for the ARP case explains, the packet is going out through the same interface it came in, so we shouldn't need `adjust_shift_l2_hdr()`. A comment explaining the why suffices.